.github/ISSUE_TEMPLATE/baseline_request.yml

```diff
@@ -0,0 +1,72 @@
+name: Baseline request
+description: Suggest a new baseline
+labels: ["new baseline, good first issue"]
+title: "Add Flower Baseline: [new_baseline_name]"
+
+body:
+  - type: input
+    attributes:
+      label: Paper
+      description: What paper would you like implemented as a baseline ?
+      placeholder: "Authors, year, title"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Link
+      description: Provide a link (ideally an `arxiv.org/abs/*` link) to the abstract of the paper.
+      placeholder: https://arxiv.org/abs/2007.14390
```

Comment:
Link to the Flower paper here

Reply:
This is already the case, except if you meant on line 17.

Reply:
Right, sorry!